services/keyboard.md

```diff
@@ -0,0 +1,39 @@
+# Keyboard
+
+    identifier: 0x18b05b6a
+    
+A keyboard that can send keystrokes.
+
+The codes for the key (selectors) is defined in the [HID Keyboard
+specification](https://usb.org/sites/default/files/hut1_21.pdf), chapter 10 Keyboard/Keypad Page, page 81.
```

Comment:
I think we should repeat the list here...

Reply:
it's like 5 pages long and anyone implementing HID will have to lookup that table.